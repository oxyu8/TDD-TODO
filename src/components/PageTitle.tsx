import * as React from 'react';

export const PageTitle = (props) => {
    return (
        <div style={{color: props.color}}>{props.title}</div>
    )
}



// export const PageTitle = () => {
//     return (
//         <div>test</div>
//     )
// }