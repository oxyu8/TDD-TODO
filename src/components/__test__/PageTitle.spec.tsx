import * as React from 'react';
import renderer from 'react-test-renderer'
import { PageTitle } from '../PageTitle'

test('test PageTitle', () => {
    const component = renderer.create(<PageTitle />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
})

