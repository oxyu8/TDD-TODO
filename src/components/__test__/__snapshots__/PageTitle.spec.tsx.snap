// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test PageTitle 1`] = `
<div
  style={
    Object {
      "color": undefined,
    }
  }
/>
`;
